src/lib/error-rate-check.js

```diff
@@ -0,0 +1,23 @@
+const nHealth = require('n-health');
+
+module.exports = (appName) => (
+	nHealth.runCheck({
+		type: 'graphiteThreshold',
+		metric: `
+			divideSeries(
+				sumSeries(
+					next.heroku.${appName}.web_*_${process.env.REGION || '*'}.express.default_route_GET.res.status.500.count
+				),
+				sumSeries(
+					next.heroku.${appName}.web_*_${process.env.REGION || '*'}.express.default_route_GET.res.status.*.count
+				)
+			)
+		`.replace(/\t|\n/g, ''),
+		threshold: 0.02,
```

Comment:
Wonder if this is quite a low threshold for a default? What's the rate for something like myFT or bronzier apps?